### PR TITLE
MODUSERBL-62 handling HTTP 422 errors

### PR DESCRIPTION
**Pull request by kostyantyn-kh**

### CODE REVIEW

**Comment by OleksiiKuzminov** [comment]
maybe we should add some tests on it?

**Comment by kurtnordstrom** [comment]
You would need to bump the interface from 4.8 to 5.0 since you're making a breaking change here.

File: src/main/java/org/folio/rest/impl/BLUsersAPI.java [hunk 1]
```diff
@@ -211,11 +211,20 @@ private void handleResponse(Response response, boolean requireOneResult,
         }
       } else {
         String message = "";
+        String errorMessage = "";
```
Comment (opochepam) [inline src/main/java/org/folio/rest/impl/BLUsersAPI.java]: variable initializing is redundant

Comment (kostyantyn-kh) [reply src/main/java/org/folio/rest/impl/BLUsersAPI.java]: fixed

**Comment by kostyantyn-kh** [comment]
> 
> 
> You would need to bump the interface from 4.8 to 5.0 since you're making a breaking change here.

Currently `mod-users-bl` provides interface version 3.5 (was 3.4)
`mod-login` provides 4.8 version (bumped from 4.7)

**Pull request merged**